### ISSUE TITLE
RFE: Hitting space on last in feed should go to next unread in next feed, not folder (regression from NNW3)

### ISSUE THREAD

**Issue by jamadden** [brentsimmons/NetNewsWire#567]
I'm coming from NetNewsWire 3.3.2. I have 343 feeds. On NNW3, those are grouped into a hierarchy of folders. I lost much of that structure in NNW5, preserving just top-level folders (but that's [another issue](
https://github.com/brentsimmons/NetNewsWire/issues/563) 😄 ). 

I use the keyboard almost exclusively to navigate (coffee cup in the other hand).

In NNW3, when I'm at the end of the last entry for a feed and I hit the spacebar, it takes me to the next unread in the next feed, always. (Simplified a bit, leaving out the scrolling behaviour, etc. The key point is that it never drops me out of a feed and onto a folder)

In NNW5, hitting spacebar on the last entry for a feed takes me to the next unread in the next feed within the same folder. If there are no more unread in that folder, it takes me *up* a level and shows me the entire next unread *folder*. Instead of a feed, it shows a folder.

I dislike this behaviour primarily because it breaks the `l` key to mark-all-unread; it marks the entire folder unread instead of just a feed (there are some feeds that I frequently use `l` on, and some feeds that I never do, within the same folder; sometimes it varies by article, when `l` can be a fast way to get to the end of the last article and move on without hitting spacebar to scroll a bunch of times). (I dislike this behaviour secondarily because I personally dislike the aggregated folder view across my feeds. One feed at a time please! This mattered even more in NNW3 where I could choose the sort order for each individual feed, but I still find it more pleasant in NNW5.)

If possible, I would like to request having the spacebar only navigate between feeds, never folders.

(Incidentally, I just confirmed that the `l` key displays the same behaviour difference: next-unread-in-next-feed in NNW3, next-unread-folder in NNW5. I would request that be changed too.)

**Comment by ednl** [comment]
Yes, always skipping folders (both with ```l``` and ```n```) is what I miss from old NNW. Please add an option to skip folders.

**Comment by ednl** [comment]
I found this issue when I search for open issues with the word "key" because in 6.0.2 the ```l``` function is broken again: #3092
